src/js/types/path_point.ts

```diff
@@ -0,0 +1,31 @@
+import { Coordinates, Degrees, Metres } from "./types"
+
+export enum ApproachTypeIdentifier {
+  LocalizerPerformanceVerticalGuidance = "LPV",
+  LocalizerPerformance = "LP",
+}
+
+export interface PathPoint {
+  area_code: string
+  airport_ident: string
+  icao_code: string
+  /// The identifier of the approach this path point is used in, such as `R36RY` or `R20`

```

Comment:
Perhaps also worthy of a JSDoc comment, so that it is visible without going to the source?